.github/workflows/ci.yml

```diff
@@ -549,10 +549,24 @@ jobs:
         run: ./mvnw -pl benchmarks/project jib:build -P starter
       - name: Build Worker Image
         run: ./mvnw -pl benchmarks/project jib:build -P worker
+  deploy-snyk-projects:
```

Comment:
❓ this only applies to stable/main pushes, the job for PRs is missing?

Reply:
There shouldn't be, I'll amend the description